cylc/uiserver/tests/test_workflows_mgr.py

```diff
@@ -88,7 +88,7 @@
     def side_effect(*_, **__):
         raise socket.error
     mocked_get_host_ip_by_name.side_effect = side_effect
-    reg, host, port, pub_port, client = await est_workflow(
+    reg, host, port, pub_port, client, result = await est_workflow(
```

Comment:
The `est_workflow` function could return a tuple with 5 elements, or a tuple with 6 elements. I've changed it to always return a 6-elements tuple, with the last one being `None` when there were only 5...